lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php

```diff
@@ -1617,10 +1322,7 @@ function attributeEqualTo($attributeName, $value, $delta = 0.0, $maxDepth = 10,
  */
 function callback($callback)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$callback". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)